src/content/ja/2020/markup.md

```diff
@@ -0,0 +1,1342 @@
+</figure>
+
+XHTML1.0以降、数がかなり減り、いくつかの標準的なもの、いくつかの難解なもの、さらにはインチキなdoctypeを含むロングテールに入ることはすでにおわかりでしょう。
+
+この結果から、2つの点が注目されます。
+
+1. <a hreflang="en" href="https://blog.whatwg.org/html-is-the-new-html5">living HTML</a>（通称「HTML5」）の発表から約10年、living HTMLは明らかに当たり前のものになりました。
+2. 生きたHTML以前のウェブは、XHTML1.0のような、次によく使われるdoctypにまだ見られます。XHTMLです。それらのドキュメントは、おそらくMIMEタイプが`text/html`のHTMLとして配信されていますが、これらの古いdoctypesはまだ死んでいません。
+
+### ドキュメントサイズ
+
+ページのドキュメントサイズとは、ネットワーク上で転送されたHTMLのバイト数のことで、圧縮が有効な場合はそれも含めて計算されます。630万のドキュメントのセットの両極端では
+
+* 1,110個のドキュメントは空です（0バイト）。
+* ドキュメントの平均サイズは49.17KBです（<a hreflang="en" href="https://w3techs.com/technologies/details/ce-gzipcompression">ほとんどの場合、圧縮されています</a>）。
+* 最大のドキュメントは61.19 _MB_ で、Web Almanacの中でも独自の分析と章立てが必要なほどです。
+
+では、この状況は一般的にどうなのでしょうか？　ドキュメントの中央値は24.65KBで、これは<a hreflang="en" href="https://httparchive.org/reports/page-weight">驚き</a>がない状態です。
+
+{{ figure_markup(
+  image="document-size.png",
+  caption="ネットワーク上で転送されたHTMLのバイト数（有効な場合は圧縮を含む）。",
+  description="ドキュメントのサイズ（バイト）はパーセンタイルで、デスクトップでは中央値が25.99KB。",
+  sheets_gid="2066175354",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQPKzFb574UnGTcfw5mcD1qR7RYHyGjQTc2hiMuYix0QoTH1DPe54Q2JucXL8bfZ6kjRoAfhk3ckudc/pubchart?oid=386686971&format=interactive",
+  sql_file="summary_pages_by_device_and_percentile.sql"
+  )
+}}
+
+### ドキュメント言語
+
+`html`開始タグの`lang`属性に2,863の異なる値を確認しました（Ethnologueによる<a hreflang="en" href="https://www.ethnologue.com/guides/how-many-languages">7,117の話されている言語</a>と比較してみてください）。[アクセシビリティ](./accessibility#language-identification)の章によると、ほとんどすべての値が有効なようです。
+
+全ドキュメントの22.36%が`lang`属性を指定していません。一般的には<a hreflang="en" href="https://www.w3.org/TR/i18n-html-tech-lang/#overall">そうすべきだ</a>という意見は多いです、ソフトウェアが最終的に<a hreflang="en" href="https://meiert.com/en/blog/lang/">言語を自動的に検出する</a>という事実を無視してドキュメントの言語は[プロトコルレベルで](https://developer.mozilla.org/ja/docs/Web/HTTP/Headers/Content-Language)指定することもできますが、これは私たちがチェックしなかったことです。
+
+ここでは、サンプルの中でもっとも人気のある（正規化された）10の言語を紹介します。HTTP Archiveは米国のデータセンターから英語の言語設定でクロールしているため、ページが書かれている言語を見ると、英語に偏っていることに注意してください。とはいえ、分析したサイトのコンテキストを示すために、`lang`属性を提示します。
+
+{{ figure_markup(
+  image="document-language.png",
+  alt="トップのHTMLのlang属性です。",
+  caption="HTMLの上位の`lang`属性です。",
+  description="クロールで使用された`lang`属性のトップ10を示す棒グラフ、デスクトップページの22.82%、モバイルページの22.36%が属性を設定しておらず、`en`はそれぞれ20.09%、18.08%、`ja`は15.17%、13.27%で使用されています。 また、`es`は4.86％、4.09％、`pt-br`は2.65％、2.84％、`ru`は2.21％、2.53％、`en-gb`は2.35％、2.19％、`de`は1.50％、1.92％、最後に`fr`は1.55％、1.43％で使用されていました。",
+  sheets_gid="2047285366",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQPKzFb574UnGTcfw5mcD1qR7RYHyGjQTc2hiMuYix0QoTH1DPe54Q2JucXL8bfZ6kjRoAfhk3ckudc/pubchart?oid=1873310240&format=interactive",
+  sql_file="pages_almanac_by_device_and_html_lang.sql"
+  )
+}}
+
+### コメント
+
+コードにコメントを付けることは一般的に良い習慣であり、HTMLコメントはユーザーエージェントによってレンダリングされることなく、HTML文書にメモを付けるためのものです。
+
+```html
+<!-- This is a comment in HTML -->
```

Comment:
As this is a comment and not actual code should we translate it?

Reply:
I agree.
I fixed.